back/app/services/sanitization_service.rb

```diff
@@ -140,7 +140,8 @@ class IframeScrubber < Rails::Html::PermitScrubber
       %r{\A(?:http(?:s?):)?//(?:www\.)?youtu(?:be\.com/(?:watch\?v=|embed/)|\.be/)([\w\-\_]*)},
       %r{\A(?:http(?:s?):)?//(?:www\.)?(?:player\.vimeo\.com/video|vimeo\.com)/(\d+)(?:|/\?)},
       %r{\A(?:http(?:s?):)?//fast.wistia.net/embed/iframe/([\w\-\_]*)(?:|/\?)},
-      %r{\A(?:http(?:s?):)?//(?:www\.)?dailymotion\.com/embed/video/?(.+)}
+      %r{\A(?:http(?:s?):)?//(?:www\.)?dailymotion\.com/embed/video/?(.+)},
+      %r{\A(?:http(?:s?):)?//media\.videotool\.dk/\?vn=([\w\-\_]*)}
```

Comment:
That applies to the other regex's as well, but are we allowing `//media.videotool.dk/?vn=foo-bar` (with the 2 leading slashes) on purpose? 

Otherwise, I would maybe suggest something like that:
```suggestion
      %r{\A(?:https?://)?media\.videotool\.dk/\?vn=[\w-]+}
```
or even
```suggestion
      %r{\A(https?://)?media\.videotool\.dk/\?vn=[\w-]+}
```
if you don't mind the first group to be a capturing group.

Reply:
Good catch. I'll edit the new whitelist entry and create ticket to do likewise for the others 👍

Reply:
I think we need the underscores in `([\w\-\_]*)` for the videotool urls though.

Reply:
here, `-` and `_` do not need to be escaped and `_` is actually part of the `\w` character set.

Reply:
Ah, gotcha. Thanks again!

Reply:
Seems like it can live without the trailing `/` before query parameters (by redirection), so we could make it optional as well.

Reply:
like this? `%r{\A(https?://)?media\.videotool\.dk?/(\?vn=)[\w-]+}`

Reply:
rather like this:
```suggestion
      %r{\A(https?://)?media\.videotool\.dk/?\?vn=[\w-]+}
```

Reply:
Ahh, _after_ the optional char(s). Got it.